diff --git a/node_modules/react-native/React/Modules/RCTUIManager.mm b/node_modules/react-native/React/Modules/RCTUIManager.mm
index e307fad..918a999 100644
--- a/node_modules/react-native/React/Modules/RCTUIManager.mm
+++ b/node_modules/react-native/React/Modules/RCTUIManager.mm
@@ -1452,7 +1452,7 @@ static void RCTMeasureLayout(RCTShadowView *view, RCTShadowView *ancestor, RCTRe
     }
     directEventTypes[eventName] = directEvents[eventName];
     if (RCT_DEBUG && bubblingEvents[eventName]) {
-      RCTLogError(
+      RCTLogWarn(
           @"Component '%@' re-registered bubbling event '%@' as a "
            "direct event",
           name,
@@ -1473,7 +1473,7 @@ static void RCTMeasureLayout(RCTShadowView *view, RCTShadowView *ancestor, RCTRe
     }
     bubblingEventTypes[eventName] = bubblingEvents[eventName];
     if (RCT_DEBUG && directEvents[eventName]) {
-      RCTLogError(
+      RCTLogWarn(
           @"Component '%@' re-registered direct event '%@' as a "
            "bubbling event",
           name,
@@ -1495,7 +1495,7 @@ static void RCTMeasureLayout(RCTShadowView *view, RCTShadowView *ancestor, RCTRe
     }
     bubblingEventTypes[eventName] = bubblingEvents[eventName];
     if (RCT_DEBUG && directEvents[eventName]) {
-      RCTLogError(
+      RCTLogWarn(
           @"Component '%@' re-registered direct event '%@' as a "
            "bubbling event",
           name,
